Readme.org

```diff
@@ -707,14 +720,33 @@
    custom-id, as the examples below:
 
    * [[Tips]].
-   * [[#my-heading][Heading]] with a =CUSTOM_ID= property.
+   * [[#my-heading][Heading]] with a =CUSTOM_ID= property.q
 
 ** Custom JS
 
    To pass custom JS code to ~Reveal.initialize~, state the code by
    ~#+REVEAL_INIT_SCRIPT~ (multiple statements are concatenated) or by
    custom variable ~org-reveal-init-script~.
 
+** Executable Source Blocks
+To allow live execution of code in some languages, enable the klipsify plugin by setting ~org-reveal-klipsify-src~ to non-nil.  Src blocks with the languages ~js~, ~clojure~, ~html~, ~python~, ~ruby~, ~scheme~, ~php~ will be executed with output shown in a console-like environment.  See the source code of ~org-reveal-src-block~ for more details.  
```

Comment:
the klipse plugin

Reply:
Fixed in https://github.com/yjwen/org-reveal/commit/24cb9276d37e1a123ae52eb58f41db150a1861d2